tests/test_app_missing_lines_realistic.py

```
@@ -7,7 +7,25 @@
+sys.path.insert(0, os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
+
+# Import the FastAPI app from app.py file
+import importlib.util
+import pathlib
+
+repo_root = pathlib.Path(__file__).parent.parent
+app_path = repo_root / "app.py"
+
+spec = importlib.util.spec_from_file_location("app_module", str(app_path))
+if spec is None or spec.loader is None:
+    raise ImportError(f"Cannot load app.py from {app_path}")
```

Ruff (0.13.1)

Avoid specifying long messages outside the exception class

(TRY003)